src/Tribe/Events/Attendee_List_Display.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+namespace Tribe\Tickets\Events;
+
+use Exception;
+use InvalidArgumentException;
+use Tribe__Tickets_Plus__Attendees_List;
+use WP_Post;
+
+class Attendee_List_Display {
+
+	private static $attendee_list_by_shortcode = 'tribe_tickets_attendee_list_triggered_by_shortcode';
```

Comment:
Make this `protected`, not `private` unless you've got a really good reason to.

Reply:
Done. Thanks. https://github.com/moderntribe/event-tickets/pull/1633/commits/6ebbdd4c31b8f157a3b409a0318893c30600d76e